tests/schema/pose.py

```diff
@@ -9,4 +9,4 @@ class Pose(Stream):
 
     def __init__(self, path):
         """Initializes the Pose stream."""
-        super().__init__(_reader.Pose(f"{path}_202_*", "/ceph/aeon/aeon/data/processed"))
+        super().__init__(_reader.Pose(f"{path}_202_*"))
```

Comment:
Now that model_root is optional, we can also remove this in tests/schema/legacypose.py